docs/UserGuide.md

```diff
@@ -38,7 +38,26 @@ _text_ | Caption for images.
 :bulb: | Indication that the following text is a tip.
 :exclamation: | Indication that the following text is important.
 
+### Command Format
+Commands are text that you can enter into TAB's command box to tell TAB which operation you would like to perform. Commands in TAB have several components and follow a general format as follows:<br/>
```

Comment:
It seems a bit odd Command Format to be here. It mentions the command box before the GUI explains it. How about moving this to right before Features?
And I think maybe can put the command format notes in this section? Could be in table form too.
wdyt @eeliana @Chesterwongz

Reply:
Maybe can move the Notes about command format to this section. Instead of one whole Notes block, can organise it as

Notes about Parameter:
- Each parameter prefix is separated from the others with a space...
- Parameters can be in any order...
-  only the last occurrence of the parameter will be taken...
- Extraneous parameters...

Command format notations:
- Words in UPPER_CASE...
- Items in square brackets are optional.
- Items in curly brackets...
- Items with …

Reply:
Okay